src/common/legacy_config_opts.h

```diff
@@ -844,26 +843,6 @@ OPTION(kinetic_user_id, OPT_INT) // kinetic user to authenticate as
 OPTION(kinetic_hmac_key, OPT_STR) // kinetic key to authenticate with
 OPTION(kinetic_use_ssl, OPT_BOOL) // whether to secure kinetic traffic with TLS
 
-
-OPTION(rocksdb_separate_wal_dir, OPT_BOOL) // use $path.wal for wal
-SAFE_OPTION(rocksdb_db_paths, OPT_STR)   // path,size( path,size)*
-OPTION(rocksdb_log_to_ceph_log, OPT_BOOL)  // log to ceph log
-OPTION(rocksdb_cache_size, OPT_U64)  // rocksdb cache size (unless set by bluestore/etc)
-OPTION(rocksdb_cache_row_ratio, OPT_FLOAT)   // ratio of cache for row (vs block)
-OPTION(rocksdb_cache_shard_bits, OPT_INT)  // rocksdb block cache shard bits, 4 bit -> 16 shards
-OPTION(rocksdb_cache_type, OPT_STR) // 'lru' or 'clock'
```

Comment:
@shinobu-x could you take this chance to update the option of `rocksdb_cache_type` in `options.cc`, to add 
```
    Option("rocksdb_cache_type", Option::TYPE_STR, Option::LEVEL_ADVANCED)
    .set_default("lru")
    .set_enum_allowed({"lru", "clock"})
    .set_description(""),
```
?